### ISSUE TITLE
NVDA announces incorrect state as "expanded" of "Popup trigger” button after collapsing its respective popup.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#14949]
### Steps to reproduce:

1. Enable NVDA and open above URL - [Frontline Worker Apps and Tools | Microsoft 365](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2f%2fwww.microsoft.com%2fen-us%2fmicrosoft-365%2fenterprise%2ffrontline&data=05%7c01%7cv-shgana%40microsoft.com%7cb77e4cd145c84c9d051308db5cf48d41%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c638205977693853400%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=Ntsnyve4bw0lEKG2dLXqnGQB4F9%2f114OEac%2Bh6OzxW4%3D&reserved=0)
in "Chrome" browser.
2. " Frontline Worker Apps and Tools | Microsoft 365" page will open.
3. Navigate to "Word”  button below “Apps and services included” heading using "Tab" key-> activate it->dialog will appear.
4. Close pop up using "esc" key or via close button of pop up.
5. Observe that NVDA announces incorrect state as "expanded" of "Word” button after collapsing its respective dialog.

### Actual behavior:
NVDA announces incorrect state as "expanded" of "Word” button after collapsing its respective dialog.
NVDA announces as:  "button expanded Word".

### Expected behavior:
NVDA shouldn’t announce incorrect state as "expanded" of "Word” button after collapsing its respective dialog.
NVDA should announce as:  "button collapsed Word".

#### Browser: Chrome

https://github.com/nvaccess/nvda/assets/93735775/ab948dca-2a07-4c1b-97df-59785f37313d

**Comment by msviolafangirl** [comment]
Yes, after pressing escape to close the dialog, NVDA does indeed say "expanded". However, once the dialog is closed and you arrow up or down to get to "word" again, it says "collapsed". This issue will be trick to solve, as it really depends on the individual browser and website design. To be honest I don't think this is a huge deal. It only says "expanded" just after the dialog is closed. It reports correctly afterwords.

**Comment by seanbudd** [comment]
Welcome @msftedad,

We are unable to process this issue as it stands. This appears to be a web authoring issue. Please let the developers that the state is set back to collapsed before the button is focused.

If you still believe this is a valid issue, we need a code sample. Provide a minimal HTML sample to reproduce this using [codepen](https://codepen.io/).

**Comment by msftedad** [comment]
@seanbudd Thank you for the suggestion. We have asked the developers for codepen and will get back to you once we receive an update.
